bullet_train/Gemfile.lock

```diff
@@ -379,7 +380,9 @@ GEM
     zeitwerk (2.6.12)
 
 PLATFORMS
+  arm64-darwin
   ruby
+  x86_64-linux
```

Comment:
I had to add some platforms to this file to get things to install locally. This file is only used in CI.